docs/introduction/domain-settings.md

```diff
@@ -0,0 +1,140 @@
+# Domain Settings
+
+During the development with the Shopsys Framework, it is possible to meet concepts - domain, multidomain, multilanguage.
+This document explains what these terms mean and how to work with them during the development of your project.
+
+## Domain, multidomain, multilanguage
+
+- **Domain** - Domain can be understood as one instance of eshop data.
+For example, just furniture can be bough on the domain shopsys-furniture.com while only electronics can be found on the domain shopsys-electro.com.
+It is still one application with one product catalogue.
+Access to these individual domains is provided through individual url addresses.
+All domains share one common administration.
+
+- **Multidomain attribute** - A value of this attribute can be set differently for each domain.
```

Comment:
I would rather say "A distinct value of this attribute can be set for each domain"